src/App/App.js

```diff
@@ -91,19 +87,12 @@ function App({ dexieCurrentUserInstance }) {
   })
 
   const apiSyncInstance = useMemo(() => {
-    if (
-      dexiePerUserDataInstance &&
-      apiBaseUrl &&
-      getAccessToken &&
-      handleSyncPullErrors &&
-      handleSyncPushErrors
-    ) {
+    if (dexiePerUserDataInstance && apiBaseUrl && getAccessToken && handleUserDeniedSyncPush) {
       return new SyncApiDataIntoOfflineStorage({
         dexiePerUserDataInstance,
         apiBaseUrl,
         getAccessToken,
-        handleSyncPullErrors,
```

Comment:
I am only ok with speculative code if the next PR is going to wrap it up and finish whatever the thing is, but now I am not sure when we are doing the pull stuff, so I am ripping that out.